docs/development.md

```diff
@@ -20,7 +20,11 @@ git submodule update --init --recursive
 
 - [Visual Studio](https://visualstudio.microsoft.com/downloads/)
   - Visual Studio 2022 is recommended; Visual Studio 2019 or newer is required.
-  - Latest Spectre-mitigated libs (via "Individual components" section of Visual Studio Installer)
+
+  during installation, ensure the following packages/components are installed:
+    - Latest Spectre-mitigated libs (via "Individual components")
+    - C++ Address Sanitizer (via "Individual components")
+    - "Desktop development with C++" (via "Workload")
```

Comment:
IMO the workload selection should come first, because I think that's the order VS prompts you to select stuff.

Reply:
Reordered